pyseir/cli.py

```diff
@@ -229,16 +127,84 @@ def run_infer_rt(state, states_only):
         "`--states` is recommended."
     ),
 )
-@click.option("--states-only", is_flag=True, help="If set, only runs on states.")
-@click.option("--output-dir", default="output/", type=str, help="Directory to deploy webui output.")
-def build_all(states, output_dir, states_only, fips):
+@click.option("--level", "-l", type=AggregationLevel)
+@click.option(
+    "--output-dir",
+    default="output/",
+    type=pathlib.Path,
+    help="Directory to deploy " "webui output.",
+)
+@click.option(
+    "--generate-api-v2",
+    default=False,
+    is_flag=True,
+    type=bool,
+    help="Generate API v2 output after PySEIR finishes",
+)
+def build_all(states, output_dir, level, fips, generate_api_v2: bool):
```

Comment:
I would love to eventually move the pyseir code into `libs/`

Reply:
We have a lot of desired improvements for such a young small code base :-/